doc/tools/SandboxTest.md

```diff
@@ -0,0 +1,24 @@
+# Using SandboxTest.ps1
+The sandbox test script is designed to allow you to test manifests locally without having to install the application to your windows installation directly. To do this, it makes use of the [Windows Sandbox](https://docs.microsoft.com/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview).
+
+First, [create a fork](https://docs.github.com/get-started/quickstart/fork-a-repo) of this repository and then [clone it](https://docs.github.com/repositories/creating-and-managing-repositories/cloning-a-repository) to your computer. Once the repository has finished cloning, you will need to open a powershell terminal since the SandboxTest script is designed to be run from the command line. In the powershell terminal which you just opened, navigate to the folder the repository was cloned into and then into the `Tools` folder. You can do this by using `cd <drive>\<path-to-parent>\winget-pkgs\Tools`. Once you are here, you should be ready to run the script. 
+
+To test a manifest, simply call the script with the full path to the manifest as an argument. This will validate the manifest, open Windows Sandbox, and attempt to install the package defined in the manifest. If you have already validated your package, you can skip the validation by adding `-SkipManifestValidation` when running the command.
+
+```raw
+.\SandboxTest.ps1 <path-to-manifest> [-SkipManifestValidation]
+```
+
+# System Requirements
+In order to use the SandboxTest script, you must have Windows Sandbox enabled. The Windows Sandbox has the following system requirements -
```

Comment:
Do these requirements need to be duplicated here? They are in the Windows Sandbox documentation. (if you feel like they do, I have no qualms except that if they change in the future we'll have to keep them in sync).

Reply:
I was thinking that when writing it. I think that being explicit about the requirements here could save some questions about why it may not be available for some people. My other thought here is that if the sandbox script ever had any additional requirements, then they could be placed here also

Reply:
Makes sense to me.